client/src/component/recommended.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import axios from 'axios';
+import Product from './product.jsx';
+import Scroll from './scroll.jsx';
+import style from '../style/style.css.jsx';
+import main from './main.module.css';
+
+class Recommended extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      products: [],
+    };
+    this.getProducts = this.getProducts.bind(this);
+  }
+
+  componentDidMount() {
+    axios.get('/2/recommendation/getInfo')
```

Comment:
Good catch pointing out this situation we need to resolve. Need to pull pid from url.